myuw/templates/handlebars/card/instructor_schedule/course_section.html

```diff
@@ -20,7 +20,7 @@ <h3 class="myuw-card-title">
                     {{# if sln}}
                         <span class="myuw-meta course-sln">SLN <a href="https://sdb.admin.uw.edu/timeschd/uwnetid/sln.asp?QTRYR={{ get_quarter_abbreviation ../quarter}}+{{ ../year }}&SLN={{ sln }}" target="_blank" data-linklabel="SLN {{ sln }}: {{ curriculum_abbr }} {{course_number}} {{section_id}}">{{ sln }}</a></span>
                     {{/if}}
-                        <span class="myuw-meta course-meeting-type c{{color_id}}Color">{{ meetings.0.type }}</span>
+                        <span class="myuw-meta course-meeting-type c{{color_id}}Color">{{ section_type }}</span>
```

Comment:
Just curious, what do these changes do? Are they needed for our class list work?

Reply:
section_type is available now. the effect is the same. but it is more clear this way.